### PR TITLE
3.0.0

### PR DESCRIPTION
**Pull request by Demonkratiy**
* refactored assets compilation due to latest webpack rules
* added non-personal environment logging

### CODE REVIEW

File: templates/package-json-template.js [hunk 1]
```diff
@@ -1,19 +1,32 @@
...
-            "resourceId": "rId0"
-        }
-    }  
-}`;
+	let res = {
+		version: templateOptions.visualData.version,
+		author: templateOptions.authorData,
+		"build environment": templateOptions.environment
+			? {
+					"tools version": `${templateOptions.environment.toolsVersion}`,
```
Comment (uve) [inline templates/package-json-template.js:7]: do not use whitespaces in the name

File: templates/package-json-template.js [hunk 2]
```diff
@@ -1,19 +1,32 @@
...
+					"node version": `${templateOptions.environment.nodeVersion}`,
+					"operating system": {
```
Comment (uve) [inline templates/package-json-template.js:9]: rename to os

File: templates/package-json-template.js [hunk 3]
```diff
@@ -1,19 +1,32 @@
...
+						platform: `${templateOptions.environment.operatingSystem}`,
+						version: `${templateOptions.environment.osVersion}`,
+						release: `${templateOptions.environment.osReleaseVersion}`,
+					},
+			  }
+			: "is not defined",
```
Comment (uve) [inline templates/package-json-template.js:15]: keep 'undefined'

**Review by uve — CHANGES_REQUESTED** [review]

File: templates/package-json-template.js (hunk 1 above)
Comment (Demonkratiy) [reply templates/package-json-template.js]: renamed

File: templates/package-json-template.js (hunk 2 above)
Comment (Demonkratiy) [reply templates/package-json-template.js]: renamed

File: templates/package-json-template.js (hunk 3 above)
Comment (Demonkratiy) [reply templates/package-json-template.js]: renamed

**Review by uve — APPROVED** [review]

**Pull request merged**